include/exadg/compressible_navier_stokes/postprocessor/pointwise_output_generator.cpp

```diff
@@ -0,0 +1,104 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#include <exadg/compressible_navier_stokes/postprocessor/pointwise_output_generator.h>
+
+#include <deal.II/numerics/vector_tools.h>
+
+#include <exadg/utilities/print_functions.h>
```

Comment:
as a minor comment: we typically follow the convention in ExaDG to list header files in a certain order, beginning with C/C++, followed by deal.II, and finally ExaDG. Would you like to adjust this?

Reply:
Shouldn't the header file corresponding to the .cpp file be the first include to ensure it is self-contained? I can change it for consistency, though.

Reply:
ok